packages/js/src/ui/components/elements/Preferences/Preferences.tsx

```diff
@@ -75,15 +78,42 @@ export const Preferences = () => {
 
 const ChannelsLabel = (props: { channels: ChannelPreference }) => {
   const style = useStyle();
-  const definedKeys = () =>
-    Object.keys(props.channels || {})
-      .filter((key) => props.channels[key as keyof ChannelPreference] !== undefined)
-      .map((key) => getLabel(key as ChannelType))
-      .join(', ');
+
+  const channelNames = () => {
+    const channels = [];
+
+    for (const key in props.channels) {
+      if (props.channels[key as keyof ChannelPreference] !== undefined) {
+        const isDisabled = !props.channels[key as keyof ChannelPreference];
+        const appearanceKey = isDisabled ? 'channelNameDisabled' : 'channelName';
```

Comment:
I don't think we need a different key here. Having the data attribute should be enough! We should also style it by targeting the data attribute.